pyproject.toml

```diff
@@ -49,7 +49,8 @@ dependencies = ["databricks-sdk>=0.27,<0.29",
                 "databricks-labs-blueprint>=0.6.0",
                 "PyYAML>=6.0.0,<7.0.0",
                 "sqlglot>=23.9,<25.1",
-                "astroid>=3.2.2"]
+                "astroid>=3.2.2",
+                "typing-extensions>=4.0.0;python_version<'3.11'"]
```

Comment:
Why do we need this? What license does it have? Why do we need to ship it with every installation?

Reply:
It comes with astroid if python < 3.11. We're adding it to make it explicit since we're using it.

Reply:
It's an optional Python.org package, see https://github.com/python/typing_extensions
I believe the licensing is already covered by current licensing ?